src/app/components/AmpIframe/index.tsx

```diff
@@ -0,0 +1,75 @@
+/** @jsxRuntime classic */
+/** @jsx jsx */
+/* @jsxFrag React.Fragment */
+import { jsx } from '@emotion/react';
+import React from 'react';
+import { Helmet } from 'react-helmet';
+import { GridItemMedium } from '#components/Grid';
+import styles from './index.styles';
+
+type Props = {
+  children: JSX.Element[];
+  className?: string;
+  width: string;
+  height: string;
+  src: string;
+};
+
+type ampMetadata = {
+  ampMetadata: {
+    imageWidth: string;
+    imageHeight: string;
+    image: string;
+    src: string;
+  };
+};
+
+const AmpHead = () => (
+  <Helmet>
+    <script
+      async
+      custom-element="amp-iframe"
+      src="https://cdn.ampproject.org/v0/amp-iframe-0.1.js"
+    />
+  </Helmet>
+);
+
+const AmpIframe = ({ children, className, width, height, src }: Props) => (
+  // @ts-expect-error Property 'amp-iframe' does not exist on type 'JSX.IntrinsicElements'
+  <amp-iframe
```

Comment:
You could do something similar to the image component by creating a custom type to remove the error message https://github.com/bbc/simorgh/blob/latest/src/app/components/Image/types.d.ts

Reply:
I've had a go at this and it did fix the error, thanks :)